### PR TITLE
Nits in the test

### PR DESCRIPTION
**Pull request by vagababov**
- make it easier to run individual case
- got before want
- nits

/assign @n3wscott @vaikas mattmoor 

Followup for #1877

### CODE REVIEW

File: reconciler/testing/reactions_test.go [hunk 1]
```diff
@@ -54,17 +56,15 @@ func TestInduceFailure(t *testing.T) {
 		resource:    "inmemorychannels",
 		wantHandled: true,
 	}, {
-		name:        "resource, wrong verb",
-		verb:        "create",
-		testSource:  imc,
-		resource:    "inmemorychannels",
-		wantHandled: false,
+		name:       "resource, wrong verb",
```
Comment (n3wscott) [inline reconciler/testing/reactions_test.go:59]: it would be better as a `map[string]struct` for the test name.

Comment (vagababov) [reply reconciler/testing/reactions_test.go:59]: No, I strongly disagree :)

**Comment by vagababov** [comment]
/retest
unrelated

**Pull request merged**